packages/components/addon/components/hds/tooltip-button/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+export default class HdsTooltipIndexComponent extends Component {
+  get text() {
+    return this.args.text;
+  }
+
+  get options() {
+    return {
+      // parse `content` strings as HTML
+      allowHTML: true,
```

Comment:
Would it be ok to leave to the consumers or do you think we should do it?

We would have to decide upon a list of elements to allow or disallow which could get messy.

Reply:
@alex-ju suggestions?

Reply:
One idea: since now we're allowing consumers to add their own options via the `extraTippyOptions`, we could do this:
- remove the `allowHTML` option from here, so _by default_ the `@text` argument is not parsed as HTML
- tell the consumers that, **if** they want to pass basic HTML they can, but **they** will have to take care of sanitizing the content/HTML

It's a win-win for me :)

Reply:
@didoo's suggestion sounds good to me – this was we're not the gatekeepers for allow/disallow. I does couple the API with the Tippy implementation (if we ever decide to swap to another library) but feels like a cleaner cut then us adding a wrapper for escaping or encoding that we'd have to maintain.

Reply:
Actually I think I'll keep the example showing HTML being used and just update it to explain it's an example of a tippy option and consumers will have to sanitize data if they use it.